src/transformers/generation/configuration_utils.py

```diff
@@ -270,6 +273,7 @@ def __init__(self, **kwargs):
         self.forced_decoder_ids = kwargs.pop("forced_decoder_ids", None)
         self.sequence_bias = kwargs.pop("sequence_bias", None)
         self.guidance_scale = kwargs.pop("guidance_scale", None)
+        self.low_memory = kwargs.pop("low_memory", False)
```

Comment:
```suggestion
        self.low_memory = kwargs.pop("low_memory", None)
```

nit -- we'd like to keep unset flags as `None`